shell/edit/secret/index.vue

```diff
@@ -61,6 +61,7 @@ export default {
     this.isProjectScoped = isProjectScoped;
 
     if (isProjectScoped) {
+      await this.$store.dispatch('management/findAll', { type: MANAGEMENT.PROJECT });
```

Comment:
Line 75 of this file fails om edit pages because the projects aren't already fetched.